src/can.rs

```diff
@@ -41,10 +41,13 @@ use crate::gpio::*;
 
 crate::embedded_hal_error!(
     CanError,
-    embedded_hal::can::Error,
-    embedded_hal::can::ErrorKind
+    embedded_hal_0_2::can::Error,
```

Comment:
I think we need something much more involved than that. Is this even working with e-hal 1.0 alpha-X?

Reply:
Oh, that's interesting. It looks like I misunderstood the versioning in Cargo.toml - embedded_hal_0_2 is pre-1.0.0, whereas embedded_hal is 1.0.0-alpha.7 (for the time being). If that's the case, feel free to close this PR; I'll try a different angle.

Reply:
It does build and work in this form as of embedded-hal 1.0.0-alpha.7.

Reply:
Yes but the user then needs to depend on both e-hal 0.2 and 1.0. He should be able to only use and depend on one of these.